.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@ the change does.
  - Docs Pull Request
 
 ##### COMPONENT NAME
-<!--- Name of the module/plugin/module/task -->
+<!--- Name of the module, plugin, module or task -->
```

Comment:
Unrelated change

Reply:
I'm sorry about that; this is my first-ever PR. I think I fixed this following the rebase instructions, but I also thought I'd done this right the first time. 😕